### PR TITLE
Add GoTOMLBear

### PR DESCRIPTION
**Pull request by Vamshi99**
This bear uses gotoml to know the type of every key value used in a file.

Closes https://github.com/coala/coala-bears/issues/1375

### CODE REVIEW

File: bears/go/GoTOMLBear.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+@linter(executable='gotoml',
+        use_stdin=True,
+        output_format='corrected')
+class GoTOMLBear:
+    """
+    It is used to the know the type of every key used in a file.
+    """
+    LANGUAGES = {'Go'}
+    REQUIREMENTS = {GoRequirement(
```
Comment (jayvdb) [inline bears/go/GoTOMLBear.py]: Can you format it like
```
REQUIREMENTS = {GoRequirement('github.com/BurntSushi/toml/cmd/tomlv',
                              flag='-u')}
```

It is helpful if the first parameter is on the same line as the `*Requirement`

However, the `-u` flag is a bit silly.  That just updates it, which isnt sensible when we have versioning.

File: .ci/deps.sh [hunk 2]
```diff
@@ -117,6 +117,7 @@ go get -u golang.org/x/tools/cmd/goimports
 go get -u sourcegraph.com/sqs/goreturns
 go get -u golang.org/x/tools/cmd/gotype
 go get -u github.com/kisielk/errcheck
+go get github.com/BurntSushi/toml/cmd/tomlv
```
Comment (jayvdb) [inline .ci/deps.sh]: no `-u` ?

File: bears/go/GoTOMLBear.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+@linter(executable='gotoml',
+        use_stdin=True,
+        output_format='corrected')
+class GoTOMLBear:
+    """
+    It is used to the know the type of every key used in a file.
+    """
+    LANGUAGES = {'Go'}
+    REQUIREMENTS = {GoRequirement(package='github.com/BurntSushi/toml/cmd/tomlv',
```
Comment (Vamshi99) [inline bears/go/GoTOMLBear.py]: @jayvdb This is the problem when I write in this format. Should I write in the previous format?

Comment (jayvdb) [inline bears/go/GoTOMLBear.py]: No.  Go back and read me previous message.

File: bears/go/GoTOMLBear.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+@linter(executable='gotoml',
+        use_stdin=True,
+        output_format='corrected')
+class GoTOMLBear:
+    """
+    It is used to the know the type of every key used in a file.
+    """
+    LANGUAGES = {'Go'}
+    REQUIREMENTS = {
+               GoRequirement(package='github.com/BurntSushi/toml/cmd/tomlv',
```
Comment (jayvdb) [inline bears/go/GoTOMLBear.py]: "package=" is not necessary.
See my comment from 3 hrs ago:
https://github.com/coala/coala-bears/pull/1404#discussion_r99460985

File: bears/go/GoTOMLBear.py [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+                             version='0.1.0')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Data type'}
```
Comment (yukiisbored) [inline bears/go/GoTOMLBear.py]: I think `Formatting` and `Syntax` is better, because the linter itself is for TOML and doesn't detect the "type"

Comment (Vamshi99) [inline bears/go/GoTOMLBear.py]: @yukiisbored Ok, I will change it to Formatting.

**Comment by Vamshi99** [comment]
This pull request is moved to https://github.com/coala/coala-bears/pull/1409

**Pull request closed without merging**